src/styles/components/pods-view.less

```diff
@@ -0,0 +1,125 @@
+.pod-instances-table {
+
+  tbody {
+    line-height: 1.8;
```

Comment:
.. and it just happens that `@table-header-cell-text-line-height` constant is `null` and it cannot be multiplied by 1.8 :(